doc/Comments.xml

```diff
@@ -343,6 +343,10 @@ use @ExampleSession:
 ]]></Listing>
 It inserts an example into the manual just as @Example would do, but all lines are commented
 and therefore not executed when the file is read.
+All lines have to start with an &AutoDoc; comment
+(<C>#!</C>) followed by exactly one space. Both the comment and the space are removed for the manual.
```

Comment:
Really *exactly one* space, not "at least one"? What about this:
```
#! gap> Print("  test\n");
#!   test
```

Reply:
Bad wording, sorry. Of course, one can put more spaces. But *exactly one* is removed, to allow what you typed.